### ISSUE TITLE
at-spi2-core: failed to connect to socket

### ISSUE THREAD

**Issue by romildo** [NixOS/nixpkgs#33744]
## Issue description

It seems that all gtk applications cannot connect to accessibility bus on unstable NixOS.

For instance, running `gedit` on a terminal I am getting:
```
$ gedit

** (gedit:11959): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-cbjLtIjt2o: Connection refused
```

The service is enabled in `configuration.nix`:
``` nix
  services.gnome3.at-spi2-core.enable = true;
```

It seems that the window manager or desktop is irrelevant, as it happens with openbox, fluxbox, enlightenment, mate, and others.

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.12, NixOS, 18.03pre125026.f607771d0f5 (Impala)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 1.11.16`
 - channels(root): `"nixos-18.03pre125026.f607771d0f5"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

**Comment by prusnak** [comment]
This does not seem to be an issue anymore. If you still see it, please reopen.
